packages/sit-onyx/src/index.ts

```diff
@@ -101,6 +104,8 @@ export * from "./components/OnyxUserMenu/types";
 export { default as OnyxVisuallyHidden } from "./components/OnyxVisuallyHidden/OnyxVisuallyHidden.vue";
 export * from "./components/OnyxVisuallyHidden/types";
 
+export { default as OnyxFlyoutMenu } from "./components/OnyxFlyoutMenu/OnyxFlyoutMenu.vue";
```

Comment:
The FlyoutMenu is already exported. Did you want to export the future instead?

Reply:
Yes, it was an error, I removed it.